scripts/cxg_admin.py

```diff
@@ -100,12 +102,15 @@ def update_collection_owner(ctx, collection_uuid, new_owner):
 @click.pass_context
 def transfer_collections(ctx, curr_owner, new_owner):
     """Transfer all collections owned by the curr_owner to the new_owner. You must first SSH into the target
-    deployment using `make db/tunnel` before running."""
+    deployment using `make db/tunnel` before running.
+    To run (from repo root)
+    ./scripts/cxg_admin.py --deployment prod transfer-collections $CURR_OWNER_ID $NEW_OWNER_ID
```

Comment:
Might add a note on how to get these IDs? Probably needs the help of an engineer to look in the database? Or look at your own API response?